bouncer.go

```diff
@@ -517,6 +525,7 @@ func handleStreamCache(bouncer *Bouncer) error {
 	}
 	bouncer.log.Debug("handleStreamCache:updated")
 	isCrowdsecStreamHealthy = true
+	bouncer.nbFailedStreamUpdate = 0
```

Comment:
do with a global var like `isCrowdsecStreamHealthy` or replace `isCrowdsecStreamHealthy` by `bouncer.isCrowdsecStreamHealthy`

Reply:
if there was no reason to use global var in the first place, i suggest add an attribute for the bouncer